index.html

```diff
@@ -13,7 +15,218 @@
 </head>
 
 <body>
-  <h1>Conference Page</h1><i class="fa-solid fa-computer"></i>
+  <div class="wrapper social-top">
+    <ul>
+      <li>
+        <a class="icon" href="https://github.com/carlosmuerto/capstone-project-conference-page">
+          <i class="fa-brands fa-github"></i>
+        </a>
+      </li>
+    </ul>
```

Comment:
- [ ] Kindly add the rest of links to match the original layout.

![image](https://user-images.githubusercontent.com/35535130/181385399-fe3ff339-baea-49dc-a70a-be7640fd014d.png)

![image](https://user-images.githubusercontent.com/35535130/181385425-f33b691d-6d35-4e3b-a5cc-7f1100dbdf70.png)